src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/PerJobConfiguration.java

```diff
@@ -24,57 +24,28 @@
 package org.jenkinsci.plugins.junitrealtimetestreporter;
 
 import hudson.Extension;
-import hudson.model.JobProperty;
-import hudson.model.JobPropertyDescriptor;
 import hudson.model.AbstractProject;
-import hudson.model.Job;
-import net.sf.json.JSONObject;
+import jenkins.model.OptionalJobProperty;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
 
-import org.kohsuke.stapler.StaplerRequest;
+public class PerJobConfiguration extends OptionalJobProperty<AbstractProject<?,?>> {
 
-public class PerJobConfiguration extends JobProperty<Job<?,?>> {
+    @DataBoundConstructor
+    public PerJobConfiguration() {}
```

Comment:
Note: changes to this class could be reverted if desired—no longer needed for Pipeline. But probably still desirable anyway.

Reply:
They will stay, thanks!